### ISSUE TITLE
请问table中的数据集STD是单纯指使用合成数据集预训练吗？还是使用STD+URD？

### ISSUE THREAD

**Issue by Echhoo** [TongkunGuan/CCD#8]
还是我应该理解成URD数据集是单纯用来做预训练的。data的：STD和ARD用来做微调，十分感谢~

**Comment by TongkunGuan** [comment]
> 还是我应该理解成URD数据集是单纯用来做预训练的。data的：STD和ARD用来做微调，十分感谢~

The pre-training experiments are conducted on URD and STD without labels, at a resolution of 32 × 128, for fair comparisons.
Our text recognition network is fine-tuned at 32×128 resolution with STD or ARD.
